app/Config/Cache.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Config;
 
+use CodeIgniter\Cache\Handlers\BaseHandler;
```

Comment:
Why is this added? I don't remember it being used.

Reply:
Oops! I was using it to reference the original value but then I used the string directly after I deprecated the constant. Good catch.